min_confidence: 0.5
image_topic: "/zed_goal/left/image_rect_color"
